control_parameters: {
  pid_parameters: {Kd: 0.2, Ki: 0.02, Kp: 9},
  power: {max: 1500.0, min: 0},
  step: {power: 500, trigger: 2},
  manual: {power: 500.0},
  automatic: {width: 140, time: 10, mode: 0}
}
